app/models/item.rb

```diff
@@ -0,0 +1,7 @@
+class Item < ApplicationRecord
+  belongs_to :merchant
```

Comment:
it is odd that items can only belong to one merchant ... I feel like multiple merchants should be able to sell the same item. I guess on Etsy everything is hand-made so it makes sense

Reply:
Think of the item itself as something unique, it cannot exist in two shops at the same time.